src/Uno.UI/UI/Xaml/UIElement.Pointers.cs

```diff
@@ -163,7 +163,7 @@ private bool IsHitTestVisibleCoalesced
 				{
 					return ctrl.IsLoaded && ctrl.IsEnabled;
 				}
-				else if (this is Windows.UI.Xaml.Controls.Control fwElt)
+				else if (this is Windows.UI.Xaml.FrameworkElement fwElt)
```

Comment:
The name suggest it was like that in the past.

Reply:
That's why I looked in the history of the file, and apparently it's more a copy paste issue (that I did since the beginning 🙄)